clients/drcachesim/tracer/tracer.cpp

```diff
@@ -427,6 +430,12 @@ event_pre_detach()
 {
     if (align_attach_detach_endpoints()) {
         NOTIFY(1, "Switching to no-tracing mode during detach\n");
+        // Keep all final thread output at the detach timestamp.  We won't add to any
+        // buffers during detach and we now have timestamps added at buffer start
+        // instead of output but a non-omitted thread that is at a wait syscall
```

Comment:
No, I mean a thread that was not completely empty and so was not omitted completely.  But this description doesn't sound right: I remember originally thinking the frozen timestamp would never actually be used, but then I saw a case but I think I mis-interpreted it.  It seems like it would only happen with a new buffer started at exit which would only happen with windows where we're now on a new window and so we make a new buffer for the thread exit.  Will update the comment.